sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/HealthApiClient.cs

```diff
@@ -73,6 +82,15 @@ public virtual async Task<Response> GetServiceStatusAsync(RequestContext context
 
         /// <summary> Get service health status. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <example>
+        /// This sample shows how to call GetServiceStatus.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new HealthApiClient(endpoint, credential);
+        /// Response response = client.GetServiceStatus();
```

Comment:
I'm wondering if we want an example if there is no input content or response body content.

Where is the service status found on the response in this call?  Ideally we would show that.

Reply:
I think the no input content case should have been handled, see those `GetXXX` operations. For no response body case, I suggest that we should print the `response.Status`, e.g., `Console.WriteLine(response.Status);`

Reply:
It looks like that is the right thing for [this API](https://docs.microsoft.com/en-us/rest/api/webpubsub/dataplane/health-api/get-service-status#response), but I wonder if it is correct for all cases.  Let's use this for now and keep an eye out if we find examples in Azure that this doesn't work for.  We can always update generated documentation after GA.